en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+        NAME                                 HOSTS                  ADDRESS     PORTS     AGE
+        api-operator-ingress-test            mgw.ingress.wso2.com   localhost   80, 443   37h
+        api-operator-ingress-pizzashackapi   mgw.ingress.wso2.com   localhost   80, 443   21h
+        
+    iii. Invoke the API using following curl command.
+    
+        curl -H "<host-name>" https://<ingress-IP>/<resource-path> -H "Authorization:Bearer $TOKEN" -k
+        
+        Example of PizzashakAPI :
+        curl -H "Host:mgw.ingress.wso2.com”  https://localhost/pizzashack/1.0.0/menu -H "accept: application/json" -H "Authorization: Bearer $TOKEN"
+        
+        Output:
+        [{"name":"BBQ Chicken Bacon","description":"Grilled white chicken, hickory-smoked bacon and fresh sliced onions in barbeque sauce","price":"18.99","icon":"/images/6.png"},{"name":"Chicken Parmesan","description":"Grilled chicken, fresh tomatoes, feta and mozzarella cheese","price":"15.99","icon":"/images/1.png"},{"name":"Chilly Chicken Cordon Bleu","description":"Spinash Alfredo sauce topped with grilled chicken, ham, onions and mozzarella","price":"23.99","icon":"/images/10.png"},{"name":"Double Bacon 6Cheese","description":"Hickory-smoked bacon, Julienne cut Canadian bacon, Parmesan, mozzarella, Romano, Asiago and and Fontina cheese","price":"27.99","icon":"/images/9.png"},{"name":"Garden Fresh","description":"Slices onions and green peppers, gourmet mushrooms, black olives and ripe Roma tomatoes","price":"12.99","icon":"/images/3.png"},{"name":"Grilled Chicken Club","description":"Grilled white chicken, hickory-smoked bacon and fresh sliced onions topped with Roma tomatoes","price":"23.99","icon":"/images/8.png"},{"name":"Hawaiian BBQ Chicken","description":"Grilled white chicken, hickory-smoked bacon, barbeque sauce topped with sweet pine-apple","price":"14.99","icon":"/images/7.png"},{"name":"Spicy Italian","description":"Pepperoni and a double portion of spicy Italian sausage","price":"11.99","icon":"/images/2.png"},{"name":"Spinach Alfredo","description":"Rich and creamy blend of spinach and garlic Parmesan with Alfredo sauce","price":"9.99","icon":"/images/5.png"},{"name":"Tuscan Six Cheese","description":"Six cheese blend of mozzarella, Parmesan, Romano, Asiago and Fontina","price":"21.99","icon":"/images/4.png"}]%
+
+### Securing APIs deployed in privateJet mode
+
+By default the APIs deployed in cloud clusters with k8s-api-operator are secured with JWT authentication. Hence the APIs deployed in privateJet mode are secured with JWT authentication.
+
+WSO2 API offers JWT authentication, Oauth2 authentication and Basic authentication mechanisms to secure your API from unauthenticated access in cloud-native API management. 
+
+In order to secure APIs with above mechanisms,
+
+   1. Create the corresponding [security custom resources and secrets](https://github.com/wso2/k8s-api-operator/blob/v1.1.0/docs/HowToGuide/OverviewOfCrds/apply-security-to-api.md)
```

Comment:
how do we do that? shall we say follow the steps in the document and create?

Reply:
Done